conda/recipes/ucx-py/meta.yaml

```diff
@@ -2,7 +2,7 @@
 
 {% set data = load_file_data("pyproject.toml") %}
 
-{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') %}
+{% set version = environ.get('RAPIDS_PACKAGE_VERSION', '0.0.0.dev').strip('"').lstrip('v') %}
```

Comment:
Will we need a different variable name/different contents for UCX-Py here?

Reply:
No, I don't think so. The variable name is generated in the build scripts in this repo, not by some external tooling, so there's no conflict issues.